cmd/argo/commands/common.go

```diff
@@ -14,27 +14,31 @@ import (
 	"k8s.io/client-go/rest"
 
 	"github.com/argoproj/argo/cmd/argo/commands/client"
+	"github.com/argoproj/argo/pkg/apiclient/workflow"
 	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
 	"github.com/argoproj/argo/pkg/client/clientset/versioned"
 	"github.com/argoproj/argo/pkg/client/clientset/versioned/typed/workflow/v1alpha1"
-	wfApiServer "github.com/argoproj/argo/server/workflow"
 	"github.com/argoproj/argo/workflow/templateresolution"
 )
 
 // Global variables
 var (
-	restConfig       *rest.Config
-	clientset        *kubernetes.Clientset
-	wfClientset      *versioned.Clientset
-	wfClient         v1alpha1.WorkflowInterface
+	// DEPRECATED
```

Comment:
Why can't these just be deleted? AFAIK, people don't import our CLI

Reply:
our CLI uses these methods - this is really more to make it easy for us to identify methods to remove